Change source of subscriber pack json

Timardo opened the pull request:
Since default subscriber.json url doesn't for some reason work for everyone, I added this little upgrade which will optionally use another source with the same database of emoticons which is even smaller than the subscriber one because it doesn't contain badges.

Should be ported to newer versions or even replace the old subscriber source, but it's not my decision.

Timardo committed:
Added option to grab subscriber emotes from different source

Since default subscriber.json url doesn't for some reason work for everyone, I added this little upgrade which will optionally use another source with same database of emoticons which is even smaller than the subscriber one because it doesn't contain badges.

blay09 commented:
Is there a reason not to use this smaller source? If not, this should probably just become the new default (though it should still support `EmoticonConfig.twitchSubscriberRegex`).

Timardo commented:
I don't know any reason why not, and yes, just after commit I noticed that I forgot on the regex thing.

Timardo committed:
meh

Rollback changes with alternate pack and add images.json as default subscriber emotes source.

Timardo commented on src/main/java/net/blay09/mods/eiramoticons/addon/pack/TwitchSmileyPack.java:
```diff
@@ -56,7 +56,7 @@ public TwitchSmileyPack(int smileySet) {
 		linkComponent.getChatStyle().setBold(true);
 		linkComponent.getChatStyle().setUnderlined(true);
-		EiraMoticonsAPI.registerEmoticonGroup("Twitch Smileys", new ChatComponentTranslation("eiramoticons:command.list.twitch.smileys", linkComponent));
+		EiraMoticonsAPI.registerEmoticonGroup("Twitch Smileys", new ChatComponentTranslation("eiramoticons:command.list.twitch.smiley", linkComponent));
```
This thing was broken, so I just fixed it
